_source/_posts/2025-02-26-okta-hosted-sign-in-widget.md

```diff
@@ -0,0 +1,836 @@
+---
+layout: blog_post
+title: "From Meh to Wow: Customize Your Okta Sign-In Experience"
+author: nesh-popovic
+by: internal-contributor
+communities: [javascript]
+description: "Tutorial on customizing your Okta Hosted Sign-In Widget to the next level."
+tags: [sign-in-custom-domain, custom-domain, okta-hosted-sign-in-widget]
+tweets:
+- ""
+- ""
+- ""
+image: blog/okta-hosted-sign-in-widget/sign-in-widget-social-image.jpeg
+type: conversion
+github: https://github.com/oktadev/okta-hosted-sign-in-widget
```

Comment:
The naming must follow our conventions. We always use `okta-<language>-<keywords>-example`